### PR TITLE
change dependencies for trivy scan step

### PR DESCRIPTION
**Pull request by macfarla**
Signed-off-by: Sally MacFarlane <sally.macfarlane@consensys.net>

small executor does not have sudo
but we don't need libSodium install in order to scan the docker image

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 version: 2.1
 orbs:
-  slack: circleci/slack@3.4.2
```
Comment (siladu) [inline .circleci/config.yml:4]: How is slack related to this PR? Just checking it's an intentional change :)

Comment (macfarla) [reply .circleci/config.yml:4]: it's not - a previous version of this CI step was attempting to use slack but it's not setup in hyperledger - so I removed that code - but forgot to remove the orb

Comment (macfarla) [reply .circleci/config.yml:4]: so yes it's an intentional change :)

**Review by siladu — APPROVED** [review]
LGTM

**Pull request merged**